netaddr.go

```diff
@@ -563,7 +559,7 @@ func (ip IP) IsMulticast() bool {
 		return ip.v4(0)&0xf0 == 0xe0
 	}
 	if ip.Is6() {
-		return ip.v6(0) == 0xff
+		return ip.addr.hi>>(64-8) == 0xff // ip.v6(0) == 0xff
```

Comment:
why not ip.v6(0) here? Did something become sad?

Reply:
the `ip.v6` AST is slightly heavier now, which pushes IsMulticast over the edge of inlineability without this (cry).

Reply:
Le sadness. I wish we could tweak the inlining costs some way other than futzing with the code :(

Reply:
Well, usual counterargument is then everybody would just say "inline everything!" and make things slower in a different way.

Reply:
Sure, knives are sharp. I still want some in my kitchen ;P